src/measure_theory/lp_space.lean

```diff
@@ -1428,6 +1449,15 @@ begin
   { exact norm_indicator_const_Lp hp_pos hp_top, },
 end
 
+lemma mem_ℒp_add {f g : α → E}
```

Comment:
Maybe change the name to the less generic
```suggestion
lemma mem_ℒp_add_of_disjoint {f g : α → E}
```

Reply:
It's a good idea, and I should probably also change the lemma `integrable_add`, from #3978, which I modelled it on.